main_app/services.py

```diff
@@ -7,9 +7,9 @@
 
 def get_parks(self, coordinates=''):
     KEY = config('KEY')
-    radius = 2000
+    radius = 48000
     if len(coordinates.split(',')[0]) > 1 and float(coordinates.split(',')[0]):
-        endpoint_url = f"https://maps.googleapis.com/maps/api/place/nearbysearch/json?fields=photos,formatted_address,name,rating,opening_hours,geometry&key={KEY}&keyword=dog+park&location={coordinates}&radius=2000"
```

Comment:
Forgot we had the radius hard corded in as 2,000, so I changed it to the variable that we assigned on line 10, then changed the value of the variable to 48000